src/System.Memory/tests/ReadOnlyBuffer/ReadOnlySequenceTests.Common.cs

```diff
@@ -24,11 +24,17 @@ public void SegmentStartIsConsideredInBoundsCheck()
 
             var buffer = new ReadOnlySequence<byte>(bufferSegment1, 0, bufferSegment2, 50);
 
-            SequencePosition c1 = buffer.GetPosition(buffer.Start, 25); // segment 1 index 75
-            SequencePosition c2 = buffer.GetPosition(buffer.Start, 55); // segment 2 index 5
+            SequencePosition c1 = buffer.GetPosition(25); // segment 1 index 75
```

Comment:
I wonder if we should not call the API "PositionAt". It would allow us to add PositionOf(byte value) in the future without having problems with inconsistent names or overloads. 

@davidfowl, @terrajobst

Reply:
That is my point. We will endup with GetPosition and PositionOf, which is inconsistent. It would be better to have either PositionAt and PositionOf or GetPositionAt and GetPositionOf.